Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -9617,6 +9640,27 @@ public void SetCurrentPageLoadStrategy(DriverOptions options)
 
         }
 
+        private void SetBrowserLogLevel(DriverOptions options)
+        {
+            if (!string.IsNullOrEmpty(BrowserLogLevel))
+            {
+                int numberLogLevel;
+                if (int.TryParse(BrowserLogLevel, out numberLogLevel))
+                {
+                    if (numberLogLevel >= 0 && numberLogLevel <= 4)
+                    {
+                        options.SetLoggingPreference(OpenQA.Selenium.LogType.Browser, (OpenQA.Selenium.LogLevel)numberLogLevel);
+                    }
+                    else throw new Exception("Please enter a valid number in the BrowserLogLevel parameter");
```

Comment:
Please update Exception message like below 
"Please enter a valid number in the BrowserLogLevel parameter in AgentConfiguration"